binderhub/base.py

```diff
@@ -13,6 +13,7 @@ def set_default_headers(self):
         headers = self.settings.get('headers', {})
         for header, value in headers.items():
             self.set_header(header, value)
+        self.set_header("access-control-allow-headers","cache-control")
```

Comment:
Can you insert a space between the two arguments `"access-control-allow-headers", "cache-control"` please?